tests/util/isFirstLetterCapitalized.js

```diff
@@ -20,4 +20,11 @@ describe('isFirstLetterCapitalized', () => {
     assert.equal(isFirstLetterCapitalized('IsCapitalized'), true);
     assert.equal(isFirstLetterCapitalized('UPPERCASE'), true);
   });
+
+  it('should return false for non-letters', () => {
```

Comment:
you haven't checked non-english language characters. see https://stackoverflow.com/a/38123020/632724